angular-instantsearch/autocomplete-results-page/README.md

```diff
@@ -1,5 +1,7 @@
 # autocomplete-results-page
 
+[![Edit autocomplete-results-page](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/algolia/doc-code-samples/tree/fix-angular-12/angular-instantsearch/autocomplete-results-page)
```

Comment:
maybe we can make that branch master? not sure why it isn't merged

Reply:
If I'm remembering correctly, the version of angular in master is newer than the one in the branch, but we found out Codesandbox only works with a version set to Angular 12 for us.

Alternatively, I could link all angular-instantsearch code samples to cloud sandboxes, which will work on master at the cost of a longer cold boot time. Wdyt?